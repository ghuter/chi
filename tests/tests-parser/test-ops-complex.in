x :: f1().x - f2()[1];
y :: f3().tab[x].fst;
