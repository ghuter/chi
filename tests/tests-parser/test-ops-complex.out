const SDECL(x, EBINOP(SUB, EACCESS(ECALL(EMEM(f1)) : ?.x) : ?, ESUBSCR(ECALL(EMEM(f2)) : ?[ECSTI(1)]) : ) : ?) : ?
const SDECL(y, EACCESS(ESUBSCR(EACCESS(ECALL(EMEM(f3)) : ?.tab) : ?[EMEM(x)]) : .fst) : ?) : ?
